File: 100/122 - Best Time to Buy and Sell Stock II.c
int maxProfit(int* prices, int pricesSize) {
    int ret = 0, i;
    
    for (i = 1; i < pricesSize; ++i)
        ret += prices[i] > prices[i - 1] ? prices[i] - prices[i - 1] : 0;
    
    return ret;
}

File: csharp/122.best-time-to-buy-and-sell-stock-ii.cs
/*
 * @lc app=leetcode id=122 lang=csharp
 *
 * [122] Best Time to Buy and Sell Stock II
 */
public class Solution {
    public int MaxProfit(int[] prices) {
        int ret = 0;
        for (int i = 1; i < prices.Length; ++i) {
            ret += Math.Max(0, prices[i] - prices[i - 1]);
        }
        return ret;
    }
}

